### PR TITLE
Add test travis file

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: C
+
+before_install:
+    - (cd ..; hg clone https://hg.mozilla.org/projects/nspr)
+
+script:
+    - make nss_build_all
+    - cd tests; NSS_TESTS=ssl_gtests NSS_CYCLES=standard ./all.sh
```
Comment (martinthomson) [inline .travis.yml]: This is going to run more than I think you really want to run.  Can you just run the ssl_gtests ?

Comment (ekr) [inline .travis.yml]: I have basically no idea what the various settings do.

Comment (martinthomson) [inline .travis.yml]: We can tune it later.

**Comment by martinthomson** [comment]
r+

**Comment by martinthomson** [comment]
We might need to find a way to reduce the amount of output this generates...

Also 
```
tstclnt: error looking up host: PR_DIRECTORY_LOOKUP_ERROR: A directory lookup on a network address has failed
```

**Comment by ekr** [comment]
Yeah, I think for now let's just restrict it to TLS.

**Pull request closed without merging**